### ISSUE TITLE
EXCEPTION occurred when running the SLUHandsOn with other dataset

### ISSUE THREAD

**Issue by hongjiedai** [Microsoft/CNTK#2160]
I tried to follow the SLUHandsOn tutorial on my dataset.
I can successfully train/test my dataset for the baseline configruation.
After adding BatchNormalizationLayer, I can successfully train my model, but fail to test the model and occurred the following error messages:

cuDNN failure 9: CUDNN_STATUS_NOT_SUPPORTED ; GPU=1 ;...; expr=cudn
nBatchNormalizationForwardInference(*m_cudnn, mode, &C::One, &C::Zero, m_inOutCu
DnnT, ptr(in), m_inOutCuDnnT, ptr(out), m_scaleBiasCuDnnT, ptr(scale), ptr(bias)
, ptr(runMean), ptr(runVariance), m_cudnnEpsilon)
Probably hitting cuDNN limit on batch size, try reducing minibatch size

[CALL STACK]
    > Microsoft::MSR::CNTK::TensorView<double>::  Reshaped
    - Microsoft::MSR::CNTK::CudaTimer::  Elapsed
    - Microsoft::MSR::CNTK::Matrix<float>::  __autoclassinit2 (x11)
    - Microsoft::MSR::CNTK::MatrixBase::  MatrixBase
    - BaseThreadInitThunk
    - RtlUserThreadStart

EXCEPTION occurred: cuDNN failure 9: CUDNN_STATUS_NOT_SUPPORTED ; GPU=1 ; ... ; expr=cudnnBatchNormalizationForwardInference(*m_cudnn, mode, &C::One,
 &C::Zero, m_inOutCuDnnT, ptr(in), m_inOutCuDnnT, ptr(out), m_scaleBiasCuDnnT, p
tr(scale), ptr(bias), ptr(runMean), ptr(runVariance), m_cudnnEpsilon)
Probably hitting cuDNN limit on batch size, try reducing minibatch size

I tried to reduce minibatch size, but still got the same errors.

How to deal with the problem?

**Comment by L-Ramos** [comment]
What is the size of your input and what minibatch size are you using? Also whats your video card?

**Comment by hongjiedai** [comment]
I used Titan X.
The training and test sets contains 29915 and 14999 sequences respectively.
I used the same configuration provided by the tutorial; 

  epochSize = 36000
  minibatchSize = 70

**Comment by L-Ramos** [comment]
Ok, so what values for minibatch have you tried?
For epochSize you can leave it =0 and it will use the whole dataset.
Can you paste your code here so we can check the BatchNormalization function?

**Comment by cha-zhang** [comment]
Closing this issue due to lack of response. Please reopen if needed.
